docs/troubleshooting.md

```diff
@@ -0,0 +1,54 @@
+# Trouble shooting
+
+This section is meant to provide guidance to recover from the most known issues you might run into. 
+
+### Did `deliver` but meant `deliver --local`
+read [deliver]() docs to understand the command
+##### Experience
+the work is gone and you cannot deliver the work again.
+
+##### Issue
+Deliver pushes the work to the remote repository on a `ready/name` branch, so it can be integrated if a server have been configured. 
+phlow will not deliver branches prefixed with `delivered` so you cannot re-deliver it. 
+
+##### Solution
+1. rename the delivered branch
+2. checkout the branch
+3. deliver your work again, nothing should stop you. 
+4. delete the remote branch
+
+```sh
+git branch -m delivered/10-my-issue 10-my-issue #1
+git checkout 10-my-issue    #2
+git phlow deliver --local   #3
+git push -d origin ready/10-my-issue #4
+```
```

Comment:
This is generally a good description. I am missing a description of the state before and after. Either a drawing or `git status` commands.

Reply:
very good feedback